src/utils/common/roundNumberToPrecision.ts

```
@@ -0,0 +1,30 @@
+/**
+ * Rounds numerical value to specified number of digits after the decimal point.
+ * If the value is smaller than specified precision (e.g. value = 0.009, decimals = 2), exponential notation is used.
+ * @param value
+ * @param decimals Number of digits after decimal comma. Must be 0..100.
+ * @param exponentialDecimals (Optional) Number of digits after decimal point in exponential notation. Must be 0..100. If absent, decimals parameter is used instead.
+ * @returns String representation of the number, either in fixed-point or in exponential notation.
+ */
+export function roundNumberToPrecision(
+  value: number,
+  decimals: number,
+  removeTrailingZeros: boolean = false,
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)